packages/git/src/browser/diff/git-diff-widget.tsx

```diff
@@ -232,7 +239,15 @@ export class GitDiffWidget extends GitNavigableListWidget<GitFileChangeNode> imp
             ref={ref => this.listView = ref || undefined}
             id={this.scrollContainer}
             files={files}
-            addDiffListKeyListeners={this.addGitDiffListKeyListeners} />;
+            addDiffListKeyListeners={this.addGitDiffListKeyListeners}
+            setListContainer={this.onListContainerReady} />;
+    }
+
+    protected onListContainerReady = (listContainerElement?: HTMLDivElement) => this.doOnListContainerReady(listContainerElement);
```

Comment:
please level the naming: `onListContainerReady` –>`setListContainer`
and if you remove the optional from the signature (requires an additional check on the caller site) you can write it like
```
protected setListContainer = (listContainer: HTMLDivElement) => this.deferredListContainer.resolve(listContainer)
```

Reply:
@jbicker, please rename `fileChangeListContainerIsReady` as well ;-)

Reply:
done